buildout.cfg

```diff
@@ -1,2 +1,171 @@
 [buildout]
-extends = profiles/prod.cfg
+develop = .
+
+# buildout options
+versions = versions
+show-picked-versions = true
+newest = false
+download-cache = ${buildout:directory}/downloads
+log-level = INFO
+
+# conda
+# offline: make sure all deps are in environment.yml
+conda-offline = true
+channel-priority = true
+override-channels = true
+conda-channels = birdhouse defaults conda-forge
+
+## extensions
+
+# use python site-packages
+# https://pypi.python.org/pypi/buildout.locallib/
+extensions = buildout.locallib
+
+# supervisor extension
+#extensions +=
+#    birdhousebuilder.recipe.supervisor
+supervisor-port = 9001
+
+parts =
+      twitcher
+      config
+      mongodb
+      gunicorn
+      supervisor
+      nginx
+      pytest
+      sphinx
+
+[environment]
+recipe = collective.recipe.environment
+
+[settings]
+prefix =  ${environment:HOME}/birdhouse
```

Comment:
Maybe defaults should be provided in the `[environment]` section for all references to `${environment:...}` ?
Not sure if buildout crashes on missing references or just blanks them out silently?

Reply:
In fact with buildout we should provide a custom.cfg than extends buildout.cfg and define any desired values. We are using environment because magpie provide some values in an env file and it is easier to inject that variables by environment. 
I will create a custom.cfg configured by environment and provide default values in the build.cfg.